### PR TITLE
Design fixups

### PR DESCRIPTION
**Pull request by ehershey**
Tweaks I found reading/trying to follow to add a check.

### CODE REVIEW

File: DESIGN.md [hunk 1]
```diff
@@ -258,10 +259,10 @@ The process for writing a checker goes as follows:
  - Decide on check "type name" (as seen in the yaml)
  - Write a check struct that implements the [Checker](https://github.com/chasinglogic/redalert/blob/master/checks/checks.go#L20) interface
  - Implement [Argable](https://github.com/chasinglogic/redalert/blob/master/checks/checks.go#L15) for your struct 
-   - Often you can just copy the FromArgs of the example check above.
```
Comment (chasinglogic) [inline DESIGN.md]: This is correct, it is referencing the ExampleCheck

File: DESIGN.md [hunk 2]
```diff
@@ -4,8 +4,8 @@
 ## Summary
 
 Redalert is a tool used for system validation, similar to ChefSpec/ServerSpec.
-It is written in Go and configured with YAML to be more familiar to those
-accustomed to working with Ansible.
+It is written in Go and configured with YAML and should be familiar to those
+accustomed to working with similarly modeled tools such as Ansible.
```
Comment (chasinglogic) [inline DESIGN.md]: Remove "similarly modeled tools such as" and this lgtm

Comment (ehershey) [reply DESIGN.md]: ah, sgtm too

File: DESIGN.md (hunk 1 above)
Comment (ehershey) [reply DESIGN.md]: Got it, I'll push something else that I think makes it clearer

Comment (chasinglogic) [reply DESIGN.md]: sgtm

**Comment by ehershey** [comment]
ok ready for another review @chasinglogic

**Pull request merged**